docs/csdk-guide.md

```diff
@@ -43,114 +43,116 @@ C++ SDK support read batch row. User can set batch by using withBatch(int batch)
+ *
+ * @param env JNIEnv
+ * @param path data store path
+ * 
+ */
+void builder(JNIEnv *env, char *path);
 ```
 
 ```
-    /**
-     * Configure the projection column names of carbon reader
-     *
-     * @param argc argument counter
-     * @param argv argument vector
-     * @return CarbonReaderBuilder object
-     */
-    jobject projection(int argc, char *argv[]);
+/**
+ * Configure the projection column names of carbon reader
+ *
+ * @param argc argument counter
+ * @param argv argument vector
+ * @return CarbonReaderBuilder object
+ */
+jobject projection(int argc, char *argv[]);
 ```
 
 ```
-    /**
-     *  build carbon reader with argument vector
-     *  it support multiple parameter
-     *  like: key=value
-     *  for example: fs.s3a.access.key=XXXX, XXXX is user's access key value
-     *
-     * @param argc argument counter
-     * @param argv argument vector
-     * @return CarbonReaderBuilder object
-     **/
-    jobject withHadoopConf(int argc, char *argv[]);
+/**
+ * build carbon reader with argument vector
+ * it support multiple parameter
+ * like: key=value
+ * for example: fs.s3a.access.key=XXXX, XXXX is user's access key value
+ *
+ * @param argc argument counter
+ * @param argv argument vector
+ * @return CarbonReaderBuilder object
+ *
+ */
+jobject withHadoopConf(int argc, char *argv[]);
 ```
 
 ```
-   /**
-     * Sets the batch size of records to read
-     *
-     * @param batch batch size
-     * @return CarbonReaderBuilder object
-     */
-    void withBatch(int batch);
+/**
+ * Sets the batch size of records to read
+ *
+ * @param batch batch size
+ * @return CarbonReaderBuilder object
+ */
+void withBatch(int batch);
 ```
 
 ```
-    /**
-     * Configure Row Record Reader for reading.
-     */
-    void withRowRecordReader();
+/**
+ * Configure Row Record Reader for reading.
+ */
+void withRowRecordReader();
 ```
 
 ```
-    /**
-     * build carbonReader object for reading data
-     * it support read data from load disk
-     *
-     * @return carbonReader object
-     */
-    jobject build();
+/**
+ * build carbonReader object for reading data
+ * it support read data from load disk
+ *
+ * @return carbonReader object
+ */
+jobject build();
 ```
 
 ```
-    /**
-     * Whether it has next row data
-     *
-     * @return boolean value, if it has next row, return true. if it hasn't next row, return false.
-     */
-    jboolean hasNext();
+/**
+ * Whether it has next row data
+ *
+ * @return boolean value, if it has next row, return true. if it hasn't next row, return false.
+ */
+jboolean hasNext();
 ```
 
 ```
-    /**
-     * read next carbonRow from data
-     * @return carbonRow object of one row
-     */
-     jobject readNextRow();
+/**
+ * read next carbonRow from data
+ * @return carbonRow object of one row
+ */
+jobject readNextRow();
 ```
 
 ```
-    /**
-     * read Next Batch Row
-     *
-     * @return rows
-     */
-    jobjectArray readNextBatchRow();
+/**
+ * read Next Batch Row
```

Comment:
Please optimize this one.

Reply:
> Please optimize this one.

thanks for review, I can not get your point,